NETProvider/src/FirebirdSql.Data.UnitTests/FbServicesTests.cs

```diff
@@ -32,13 +32,13 @@
 
 namespace FirebirdSql.Data.UnitTests
 {
-	[TestFixture]
+	[TestFixture(FbServerType.Default)]
```

Comment:
I had doubts about services. 
There are some tests that I think doesn't have sense for embedded connection. I.e. Shutdown, all user, properties tests.
I can extract default connection only tests to separate fixture and enable the rest with embedded connection.
What do you think?